code/__DEFINES/jobs.dm

```diff
@@ -147,6 +149,7 @@ GLOBAL_LIST_INIT(jobs_fallen_marine, typecacheof(list(/datum/job/fallen/marine),
 #define XP_REQ_INTERMEDIATE 180
 #define XP_REQ_EXPERIENCED 600
 #define XP_REQ_EXPERT 900
+#define XP_REQ_TADPOLE_OFFICER 3000
```

Comment:
Remove the new define and set it to expert, we will increase the define after to 20/25h

This was agreed on in maintainer chat